src/_guides/language/language-tour.md

```diff
@@ -3547,6 +3554,65 @@ enum Color { red, green, blue }
   to help prevent copy-paste errors.
 {{site.alert.end}}
 
+#### Declaring enhanced enums
+
+Dart also allows enum declarations to declare classes
+with fields, methods, and const constructors
+which are limited to a fixed number of known constant instances.
+
+To declare an enhanced enum,
+follow a syntax similar to normal [classes](#classes),
+but with a few extra requirements:
+
+* Instance variables must be `final`, 
+  including those added by [mixins](#mixins).
+* All constructors must be
+  [constant generative constructors](#constant-constructors).
+* No other class can be extended, [`Enum`] is the superclass.
```

Comment:
(Nitpickily, if you have a mixin, your *superclass* is not the class in the `extends` clause, but the result of the mixin applications. I'm not sure anybody actually thinks that way, though, so this is probably close enough to correct and easy to understand.)

Reply:
I appreciate the insight here, I guess I didn't really think that way before either. I've instead adjusted it to not mention the superclass.